builtin/providers/google/resource_compute_instance.go

```diff
@@ -58,6 +58,12 @@ func resourceComputeInstance() *schema.Resource {
 							ForceNew: true,
 						},
 
+						"nvme": &schema.Schema{
```

Comment:
@danawillow Any thoughts about how to test it?
I have tried but the the `TestAcc*` tests always fails on my machine